# COMPFEST (compfest-uppercase)

All strings of /com(p)?fest/gi should be COMPFEST


## Rule Details

This rule aims to identify all spellings of COMPFEST that is not COMPFEST.

Examples of **incorrect** code for this rule:

```js

const message = 'Welcome to Compfest!'

```

Examples of **correct** code for this rule:

```js

const message = 'Welcome to COMPFEST!'

```

## When Not To Use It

Do not use this rule if you don't like COMPFEST.
